docs/concepts/abstractions/overview.md

```diff
@@ -0,0 +1,58 @@
+---
+---
+
+{% capture overview %}
+This page explains how Kubernetes objects are represented in the Kubernetes API, and how you can express them in `.yaml` format.
+{% endcapture %}
+
+{% capture body %}
+### Understanding Kubernetes Objects
+
+*Kubernetes Objects* are persistent entities in the Kubernetes system. Kubenetes uses these entities to represent the state of your cluster. Specifically, they can describe:
+
+* What containerized applications are running (and on which nodes)
+* The resources available to those applications
+* The policies around how those applications behave, such as restart policies, upgrades, and fault-tolerance
+
+When you create a Kubernetes object, you create a "record of intent"--once you create the object, the Kubernetes system will constantly work to ensure that the entity exists. By creating an object, you're effectively telling the Kubernetes system what you want your cluster to be doing; this is your cluster's **desired state**.
+
+To work with Kubernetes objects--whether to create, modify, or delete them--you'll need to use the [Kubernetes API](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api-conventions.md). When you use the `kubectl` comamnd-line interface, for example, the CLI makes the necessary Kubernetes API calls for you; you can also use the Kubernetes API directly in your own programs.
+
+#### Object Spec and Status
+
+Every Kubernetes object has two major nested object fields: the object *spec* and the object *status*. The *spec*, which you must provide, describes your *desired state* for the object--the characteristics that you want the object to have. The *status* describes the *actual state* for the object, and is supplied by the Kubernetes system. At any given time, the [Kubernetes Control Plane](/docs/concepts/control-plane/overview/) actively maintains an object's actual state to match the desired state you supplied.
```

Comment:
To avoid getting too wordy and potentially confusing, I added an example using Deployment in a second paragraph. This should hopefully make the relationship clear.